packages/react/src/slider/root/SliderRoot.test.tsx

```diff
@@ -49,7 +49,7 @@ function TestSlider(props: SliderRoot.Props) {
   );
 }
 
-function TestRangeSlider(props: SliderRoot.Props) {
+function TestRangeSlider(props: SliderRoot.Props<number[]>) {
```

Comment:
@seloner I think we can define Props without using generics.

Check https://github.com/mui/base-ui/pull/1241/files#r1899037802, please